pydecred/txscript.py

```diff
@@ -493,7 +493,7 @@ def next(self):
 
         # The only remaining case is an opcode with length zero which is
         # impossible.
-        raise Exception("unreachable")
+        assert False, "unreachable"
```

Comment:
Same question about assert!

Reply:
I could see this being an assert. The stack overflow bit does say one use case is making the code explode/panic for some extraordinary reason.